resources/lang/en/api.php

```diff
@@ -3,12 +3,17 @@
 return [
 
     'messages' => [
+        'voucher_success' => '[xXx] Voucher is valid',
+        'batch_voucher_submit' => 'Thanks! Your queue has been successfully submitted. :success_amount voucher(s) accepted, :duplicate_amount duplicate(s) and :invalid_amount invalid.',
```

Comment:
No it doesn't actually get displayed. I think there is a comment to that effect in the other file, but maybe put it here too.